contracts/voting/Voting.sol

```diff
@@ -0,0 +1,350 @@
+// SPDX-License-Identifier: Apache 2.0
+// Copyright © 2021 Anton "BaldyAsh" Grigorev. All rights reserved.
+
+pragma solidity ^0.8.0;
+
+import "../common/ownership/Ownable.sol";
+import "../nft/NFT.sol";
+import "../vote/Vote.sol";
+import "../voucher/Voucher.sol";
+import "./IVoting.sol";
+import "./VotingConstants.sol";
+import "./VotingStructs.sol";
+import "./VotingErrors.sol";
+
+contract Voting is IVoting, Ownable {
+    uint256 internal constant VOTING_DURATION = 1 days;
+    uint256 internal constant GOVERNANCE_DURATION = 6 days;
+
+    uint256 internal constant PRIZE_RATE = 87;
+    uint256 internal constant REWARD_BURN_RATE = 3;
+
+    NFT internal _mayor;
+    Vote internal _voteToken;
+    Voucher internal _voucherToken;
+
+    // amount of votes for 1 citizen, 4 digits
+    uint256 internal _votesPerCitizen;
+    uint256 internal _voteDigits;
```

Comment:
uint8 is enough. you can place it before _votesPerCitizen and it will share the slot with _voucherToken

Reply:
done